flaskbb/templates/management/overview.html

```diff
@@ -138,14 +138,14 @@ <h4>{% trans %}Everything seems alright.{% endtrans %}</h4>
                                 </div>
                                 <div class="value pull-right">
                                     {% if not plugin.enabled %}
-                                        <span class="text-danger">not enabled</span>
+                                        <span class="text-danger">{% trans %}not enabled{% endtrans %}</span>
                                     {% elif plugin.enabled %}
                                         {% if plugin.is_installed %}
-                                        <span class="text-success">enabled &amp; installed</span>
+                                        <span class="text-success">{% trans %}enabled &amp; installed{% endtrans %}</span>
```

Comment:
Do we want to keep the ampersand or replace it with and? I know that seems like a nitpick but it might make other translations easier.

Reply:
Mhh not sure. Imho it wouldn't look as good as an '&'

Reply:
I'm fine either way. Just a thought